app/templating/AssetHelper.scala

```diff
@@ -95,10 +95,10 @@ if (window.matchMedia('(prefers-color-scheme: dark)').media === 'not all')
     ContentSecurityPolicy(
       defaultSrc = List("'self'", assetDomain.value),
       connectSrc =
-        "'self'" :: assetDomain.value :: sockets ::: env.explorerEndpoint :: env.tablebaseEndpoint :: localDev,
+      "'self'" :: "data:" :: assetDomain.value :: sockets ::: env.explorerEndpoint :: env.tablebaseEndpoint :: localDev,
       styleSrc = List("'self'", "'unsafe-inline'", assetDomain.value),
       frameSrc = List("'self'", assetDomain.value, "www.youtube.com", "player.twitch.tv"),
-      workerSrc = List("'self'", assetDomain.value),
+      workerSrc = List("'self'", assetDomain.value, "blob:"),
       imgSrc = List("data:", "*"),
       scriptSrc = List("'self'", assetDomain.value),
```

Comment:
I'm noticing that the reason that voice stuff won't work in round is that it has a more restrictive CSP policy, and doesn't allow 'unsafe-eval' like the puzzle page does. I don't think we want to enable that on the round page, but not sure that I see away forward without doing so, since it seems that WASM requires eval 🤔